scalableminds/webknossos
# only show team names of own organization

### Steps to test:
- create second organization, set a dataset to public
- when logged in as someone from the non-owning organization, allowed teams should be hidden
- also fixed: when showing “users with access” list in this case, no error is thrown. list should be empty.

### Issues:
- fixes #3924 

------
